relay-server/src/services/project_cache.rs

```diff
@@ -196,6 +196,11 @@ impl UpdateSpoolIndex {
 #[derive(Debug)]
 pub struct SpoolHealth;
 
+pub struct SpoolHealthWrapper {
+    pub message: SpoolHealth,
+    pub sent_instant: Instant,
```

Comment:
I really don't like this metric, this is adds a lot of changes just to carry some instant through a bunch of layers.

We already have the time how long it takes for the "answer" and we already have a timer which tells us how long the operation took. If they don't match up, the remaining time was the delay.